src/applications/claims-status/components/AppealEventItem.jsx

```diff
@@ -1,51 +0,0 @@
-/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
-import PropTypes from 'prop-types';
-import React from 'react';
-import moment from 'moment';
-import classNames from 'classnames';
-
-import { appealStatusDescriptions } from '../utils/appeal-helpers';
-
-export default class AppealEventItem extends React.Component {
-  constructor(props) {
-    super(props);
-    this.toggle = this.toggle.bind(this);
-    this.state = { open: false };
-  }
-
-  toggle() {
-    // eslint-disable-next-line react/no-access-state-in-setstate
-    this.setState({ open: !this.state.open });
-  }
-
-  render() {
-    const { event } = this.props;
-    const { status } = appealStatusDescriptions(event);
-
-    const expandIconClass = classNames({
-      'fa claims-status-icon event-header-icon': true,
-      'fa-minus': this.state.open,
-      'fa-plus': !this.state.open,
-    });
-
-    return (
-      <li className="event-item" onClick={this.toggle} onKeyPress={this.toggle}>
-        <div className="event-header">
-          <i className="fa fa-check-circle claims-status-icon event-header-icon" />
```

Comment:
That's a good point. This is one of a series of PRs that I intend to make to remove the Claim Status Tool V1 code. My focus has been on trying to make the PRs as focused as possible so that they are easy to review, as I'm the only developer on my team that works on this particular application. I'm planning on doing an audit of all of the CSS in a follow up PR